File: functions/wipe_db.js
const admin = require('firebase-admin');
admin.initializeApp({ projectId: 'algoworks-dev' });
const db = admin.firestore();

async function wipeStudents() {
    console.log("Starting aggressive wipe of 'students' collection...");
    const snapshot = await db.collection('students').get();
    
    if (snapshot.empty) {
        console.log("Collection is already empty.");
        return;
    }
    
    let batch = db.batch();
    let count = 0;
    
    for (const doc of snapshot.docs) {
        batch.delete(doc.ref);
        count++;
        if (count % 500 === 0) {
            await batch.commit();
            batch = db.batch();
        }
    }
    
    if (count > 0) {
        await batch.commit();
    }
    
    console.log(`Successfully wiped ${count} legacy dummy records.`);
}

wipeStudents().catch(console.error);

File: python-agent/wipe_db.py
from google.cloud import firestore

def wipe_students():
    db = firestore.Client(project="algoworks-dev")
    docs = db.collection('students').stream()
    
    count = 0
    for doc in docs:
        doc.reference.delete()
        count += 1
        
    print(f"Successfully destroyed {count} legacy student records.")

if __name__ == '__main__':
    wipe_students()
